config/initializers/sidekiq.rb

```diff
@@ -1,6 +1,6 @@
 Sidekiq.configure_server do |config|
   config.redis = EmailAlertAPI.config.redis_config
-  config.error_handlers << Proc.new {|ex, context_hash| Airbrake.notify(ex, context_hash) }
```

Comment:
The Sentry client does error reporting for Sidekiq by itself, so this can be removed.

Reply:
Even better!